packages/ods-react/src/components/pagination/src/components/pagination-page-size-selector/PaginationPageSizeSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { usePaginationContext } from '@ark-ui/react/pagination';
+import { type FC } from 'react';
+import { Select, SelectContent, SelectControl, type SelectItem } from '../../../../select/src';
+import { PAGINATION_PER_PAGE_OPTIONS } from '../../constants/pagination-per-page';
+import style from './paginationPageSizeSelector.module.scss';
+
+type RenderTotalItemsLabel = (params: { totalItems: number }) => string | number;
```

Comment:
As we expose all at the root of the package (for now), we need to prefix everything, something like `PaginationTotalItemsLabelRenderer`